code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -102,7 +105,7 @@
 	icon = stored_mmi.icon
 	icon_state = stored_mmi.icon_state
 
-/obj/item/organ/mmi_holder/removed(var/mob/living/user)
+/obj/item/organ/internal/brain/mmi_holder/remove(var/mob/living/user)
```

Comment:
Careful, this might potentially have issues of qdeling the brain with the brainmob still "inside" of it and leaving you with a brainmobless MMI